src/manipulation.ts

```diff
@@ -8,4 +10,27 @@ export const mapPropertyDown = (source: string, target: string, defaultValue?: a
 export const mapPropertyUp = (source: string, target: string, defaultValue?: any) => async (ctx, next?) => {
   next && await next()
   ctx.state[target] = get(ctx.state, source, defaultValue || null)
+}
+
+export const putInState = async (variableName: string) => {
```

Comment:
Since it's from event, could we call it `eventToState` ? Not obvious, cause it could be from context